Alternate method for moving the sync folder (#92)
owncloud/docs-client-desktop: modules/ROOT/pages/faq.adoc

:toc: right
:toclevels: 1

== Some Files Are Continuously Uploaded to the Server, Even When They Are Not Modified

> It is possible that another program is changing the modification date of the file. If the file is uses the `.eml` extension, Windows automatically and continually changes all files, unless you remove. 

Reviewer: ```suggestion
It is possible that another program is changing the modification date of the file. If the file has an `.eml` extension, Windows automatically and continually changes the file, unless you remove it. 
```

+
image:setup/ownCloud-remove_existing_connection_confirmation_dialog.png[image,width=60%]

..  Add a new connection which syncs to the desired directory.
+
Click the drop-down menu menu:Account[Add new].
+
This opens the ownCloud Connection Wizard, which you can see below, _but_ with an extra option. This option provides the ability to either keep the existing data _(synced by the previous connection)_ or to start a clean sync _(erasing the existing data)_.
+
[IMPORTANT]
====
Be careful before choosing the "Start a clean sync" option. The old sync folder _may_ contain a considerable amount of data, ranging into the gigabytes or terabytes. If it does, after the client creates the new connection, it will have to download *all* of that information again.

Instead, first move or copy the old local sync folder, containing a copy of the existing files, to the new location. Then, when creating the new connection choose "_keep existing data_" instead. The ownCloud client will check the files in the newly-added sync folder and find that they match what is on the server and not need to download anything.
====
+
image:setup/ownCloud-replacement_connection_wizard.png[image,width=60%]
+

> Make your choice and click btn:[Connect...] This will then step you through the Connection Wizard, just as you did when you setup the previous sync connection, but giving you the opportunity to choose a new sync directory.

Reviewer: ```suggestion
Make your choice and click btn:[Connect...] This will then lead you through the Connection Wizard, just like when you set up the previous sync connection, but giving you the opportunity to choose a new sync directory.
```
